websocket/backend_server/README.md

```diff
@@ -0,0 +1,29 @@
+## Ballerina with Websockets
+
+Modern web and mobile app users expect real-time updates, whether it's tracking a cab's live location or viewing up-to-the-minute inventory levels. In-app chat functionality with sales agents or colleagues has also become a common feature. WebSockets emerge as the ideal technology for these real-time data transfers between front-end and back-end systems. 
```

Comment:
```suggestion
Modern web and mobile app users expect real-time updates, whether it's tracking a cab's live location or viewing up-to-the-minute inventory levels. In-app chat functionality with sales agents or with colleagues has also become a common feature. WebSockets emerge as the ideal technology for these real-time data transfers between front-end and back-end systems. 
```

Reply:
@praneesha can you check this?

Reply:
Repeatign the word `with` is redundant here.